website/js/DataTable.js

```diff
@@ -37,22 +38,31 @@ function setPages({data, count, deletedCount, synonyms, releaseName}, pageLength
 // Wrap Table with a version having PureRenderMixin
 var FastTable = React.createClass({
     mixins: [PureRenderMixin],
-    truncateGenomicCoordinates: function(variantData) {
-        const genomicCoordinateFields = ["Genomic_Coordinate_hg37", "Genomic_Coordinate_hg38"];
-        genomicCoordinateFields.forEach(function(field) {
+    normalizeGenomicCoordinate: function(field, hgvs, variantData) {
+        if (!util.isEmptyField(hgvs)) {
+            variantData[field] = hgvs;
+        } else {
             if (variantData[field].length > 35) {
                 variantData[field] = variantData[field].substring(0, 35) + "...";
-            } else {
-                return variantData[field];
             }
-        });
+        }
+        return variantData;
+    },
+    determineGenomicCoordinates: function(variantData) {
+        let field = "Genomic_Coordinate_hg37";
+        let hgvs = variantData.Genomic_HGVS_37;
+        variantData = this.normalizeGenomicCoordinate(field, hgvs, variantData);
+
+        field = "Genomic_Coordinate_hg38";
+        hgvs = variantData.Genomic_HGVS_38;
+        variantData = this.normalizeGenomicCoordinate(field, hgvs, variantData);
```

Comment:
not sure what the intention is, but it seems the `Genomic_Coordinate_hg37` related variables are just overwritten

Reply:
exactly -- the purpose of this PR is to introduce a new standardized variant nomenclature and use it if it's available. if not, we use the old Genomic_Coordinate_hg38/Genomic_Coordinate_hg37 values.